docs/README.md

```diff
@@ -139,7 +124,8 @@ You will also be able access values related to the product specifications throug
   }
 }
 ```
-In this case, `App Data` is the specificationGroup name and `vtex-url` is the specification name.
+
+In the example above, `App Data` is the specification group name and `vtex-url` is the product specification name.
```

Comment:
I would change the above example to a more common one:

`{specificationGroups.Design.specifications.Dimensions}`